requirements/requirements-testing.txt

```diff
@@ -2,3 +2,6 @@ coverage == 5.1
 # django_coverage_plugin
 sqlparse
 sqlalchemy
+funcsigs==1.0.2
+factory-boy==2.12.0
+mock==3.0.1
```

Comment:
What's funcsigns and mock libraries for?

Reply:
Resolved in [#51b12dc](https://github.com/djangoplicity/djangoplicity/pull/96/commits/51b12dc847fc89988959c6c94d0a40b700743c6b)